### PR TITLE
fix: add SR-ZG9002KR12-Pro expose and add ZG9380A zigbeeModel for SR-ZG9042MP

### PR DESCRIPTION
**Pull request by niracler**
1.  add SR-ZG9002KR12-Pro expose
2. add ZG9380A zigbeeModel for SR-ZG9042MP

### CODE REVIEW

File: src/devices/sunricher.ts [hunk 1]
```diff
@@ -226,15 +224,17 @@ const fzLocal = {
                     16: 'k12',
                 };
 
+                const action_buttons: string[] = [];
```
Comment (Koenkk) [inline src/devices/sunricher.ts:227]:
```suggestion
                const actionButtons: string[] = [];
```

File: src/devices/sunricher.ts [hunk 2]
```diff
@@ -226,15 +224,17 @@ const fzLocal = {
...
                         const button = i + 1;
-                        buttons.push(specialButtonMap[button] ?? `k${button}`);
+                        action_buttons.push(specialButtonMap[button] ?? `k${button}`);
                     }
                 }
+                return {action, action_buttons};
             } else if (messageType === 0x03) {
                 const directionMask = bytes[4];
-                speed = bytes[6];
+                const action_speed = bytes[6];
```
Comment (Koenkk) [inline src/devices/sunricher.ts:237]:
```suggestion
                const actionSpeed = bytes[6];
```

**Comment by niracler** [comment]
I resolved all conversations @Koenkk

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**